src/App.vue

```diff
@@ -2,16 +2,25 @@
     <div id="app">
         <masthead></masthead>
         <transition name="fade"><router-view></router-view></transition>
+        <privacy-link v-if="footerShow"></privacy-link>
     </div>
 </template>
 
 <script>
 import Masthead from "./components/Masthead.vue";
+import PrivacyLink from "./components/PrivacyLink.vue";
 export default {
     name: "app",
     components: {
         Masthead,
+        PrivacyLink
     },
+    computed: {
+        footerShow: function () {
+            const paths = ["/", "/scan"];
+            return !(paths.indexOf(this.$route.path) === -1);
```

Comment:
nb.
if we ever to es2016, it has `[].inlcudes()`

Reply:
not supported by IE - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility